web/src/components/feature/CalculateFluid.tsx

```diff
@@ -20,14 +20,16 @@ const FluidPackage = styled(FlexContainer)`
   }
 `
 
-export const CalculateFluid = () => {
+export const CalculateFluid = ({ edit }: { edit: () => void }) => {
   const fluidPackages = ['Krafla']
 
   const InputParameters = () => (
     <Form as="form">
       <FluidPackage>
         <Autocomplete label="Fluid package" options={fluidPackages} autoWidth />
```

Comment:
Not really this PR, but this autocomplete thing is freaking annoying.

Reply:
Agree. :((